doc/man7/provider-keyexch.pod

```diff
@@ -29,6 +29,8 @@ provider-keyexch - The keyexch library E<lt>-E<gt> provider functions
                        size_t outlen);
 
  /* Key Exchange parameters */
+ int OP_keyexch_get_params(const OSSL_PARAM params[]);
```

Comment:
this should drop `const`, right?

Reply:
Yes. This is a copy-n-paste mistake, I copied the set_ctx_param function and forgot to unconstify.

Reply:
Fixed